core/controllers/reader_test.py

```diff
@@ -500,10 +500,8 @@ def _get_recommendation_ids(
                 exploration_id, author_recommended_ids_str, collection_id_param,
                 include_recommendations_param))
 
-        response = self.get_html_response('/explore/%s' % exploration_id)
-        csrf_token = self.get_csrf_token_from_response(response)
-        summaries = self.get_json(
-            recommendations_url, params=csrf_token)['summaries']
+        self.get_html_response('/explore/%s' % exploration_id)
```

Comment:
Why do you need this call?

Reply:
For checking status code and content-type?

Reply:
Hm. The original purpose of the call was to get the response so you can get the CSRF token so that you can make the subsequent call.

If you don't need the CSRF token then do you actually need this?

Reply:
Hmm I think you are right we don't need this now.

Reply:
done